autoload/vader/window.vim

```diff
@@ -38,13 +38,16 @@ function! s:switch_to_workbench()
   execute 'b' s:workbench_bfr
 endfunction
 
-function! vader#window#open()
+function! vader#window#close()
   silent! bd \[Vader\]
-  silent! bd \[Vader-workbench\]
+  silent! bw! \[Vader-workbench\]
```

Comment:
Sure. I use vader to test https://github.com/albfan/ag.vim. It needs a real file to do searches, as it relies on native ag search https://github.com/ggreer/the_silver_searcher

As documentation says 

http://vimdoc.sourceforge.net/htmldoc/windows.html#:bd

> If the buffer was changed, this fails

I guess addding ! will be enough but It doesn't remove "Vader workbench". I try buffer wipeout and it works.

I prefer to do this, rather to open another issue related with the need to add "Given" to the test (which maybe solve this too) Older versions of vader didn't force me to add Given clause, but opens a buffer too.

https://github.com/albfan/ag.vim/blob/master/t/01-aggroup.vader#L3
https://github.com/albfan/ag.vim/blob/master/t/01-aggroup.vader#L49

Let me know if makes sense to work on this, anyway any test with unsaved touched buffers should fails to close all vader stuff. 

Should I create an expected failed test for vader using vader closing its stuff, and a passed test after use "bw!" to detect corner cases? Let me know